src/structures/ApplicationCommand.js

```diff
@@ -178,7 +184,8 @@ class ApplicationCommand extends Base {
       // Future proof for options being nullable
       // TODO: remove ?? 0 on each when nullable
       (command.options?.length ?? 0) !== (this.options?.length ?? 0) ||
-      (command.defaultPermission ?? command.default_permission ?? true) !== this.defaultPermission
+      (command.defaultPermission ?? command.default_permission ?? true) !== this.defaultPermission ||
+      command.version !== this.version
```

Comment:
This should be the same as the first checks:
```suggestion
      ('version' in command && command.version !== this.version)
```

Also, shouldn't we move this check with description and others?

Reply:
I think this check should be moved right after id, in a standalone form. If versions are equal there's no reason to check the rest

Reply:
that sounds right. btw why are we using `in` keyword even for fields that should always be present, why not use optional chaining instead 👀

Reply:
because purposefully setting a value to undefined or null (though technically invalid) causes the equal check to happen properly